pygeo/DVGeometry.py

```diff
@@ -690,6 +693,98 @@ def addGeoDVLocal(self, dvName, lower=None, upper=None, scale=1.0,
 
         return self.DV_listLocal[dvName].nVal
 
+    def addGeoDVSpanwiseLocal(self, dvName, spanIndex, axis='y', lower=None, upper=None,
+                             scale=1.0, pointSelect=None, volList=None, config=None):
+
+        if type(config) == str:
+                    config = [config]
```

Comment:
From the doc stings it looks like you can use it to control which design variables are applied to a particular config. For example you could have a landing config where you add trailing edge morphing.

config : str or list
            Define what configurations this design variable will be applied to
            Use a string for a single configuration or a list for multiple
            configurations. The default value of None implies that the design
            variable appies to *ALL* configurations.

Reply:
I see, I was just not really following the branches of the code where `config` is not the default `None`, I will take a better look. Are we testing this feature? :) 
Anyway, what you say makes sense, but now I am wondering how this is used in a run - i.e. it looks like you can use config to define a DV for a specific `setName` in our multipoint optimization, instead of relying on different DV definitions for different procs as we did for morphing and multi element airfoils in the past. This could make things much easier to set up.

Anyway, this digression is not relevant for this PR so we can talk offline, don't worry about this.